interface/reports/encounters_report.php

```diff
@@ -193,6 +193,16 @@ function dosort(orderby) {
         function refreshme() {
             document.forms[0].submit();
         }
+
+        // Called to switch to the specified encounter having the specified DOS.
+        function toEncounter(pid, pubpid, pname, enc, datestr, dobstr) {
+            encurl = 'patient_file/encounter/encounter_top.php?set_encounter=' + encodeURIComponent(enc) +
+                '&pid=' + encodeURIComponent(pid);
+            parent.left_nav.setPatient(pname, pid, pubpid, '', dobstr);
+            parent.left_nav.setEncounter(datestr, enc, 'enc');
+            parent.left_nav.loadFrame('enc2', 'enc', encurl);
+        }
```

Comment:
check out the toPatient() function in the patient_tracker.php script where it seems like that process does everything that is needed to populate a new pt/encounter. Would try something like:
```
top.restoreSession();
top.RTop.location = "<?php echo $GLOBALS['webroot']; ?>/interface/patient_file/summary/demographics.php?set_pid=" + encodeURIComponent(newpid) + "&set_encounterid=" + encodeURIComponent(enc);
```

Reply:
btw, don't ask me how it works :)